### PR TITLE
Autocomplete: add section-*

### PR DESCRIPTION
**Pull request by estelle**
Added information about the `section-*` token of the autocomplete attribute.
rearranged stuff to enable a "description" since the description was longer than an intro statement.
we need to document everything , but I did NOT add an example in this PR as this use case is rare and i just wanted to get this missing feature added.

### CODE REVIEW

File: files/en-us/web/html/attributes/autocomplete/index.md [hunk 1]
```diff
@@ -11,17 +11,29 @@ The HTML `autocomplete` attribute lets web developers specify what if any permis
...
 > **Note:** In order to provide autocompletion, user-agents might require `<input>`/`<select>`/`<textarea>` elements to:
 >
 > 1. Have a `name` and/or `id` attribute
 > 2. Be descendants of a `<form>` element
 > 3. The form to have a {{HTMLElement("input/submit", "submit")}} button
 
-{{EmbedInteractiveExample("pages/tabbed/attribute-autocomplete.html", "tabbed-shorter")}}
+If the same list of tokens is used in more than one form control, the user-agent will autocomplete all occurrences of the same `autocomplete` value with the same data value.
+
+Some tokens may be used more than once with potentially different expected values, such as the `zip code` token in a form that contains both shipping and billing addresses. By including multiple tokens in a space-separated list, the values are unique: in this case, `autocomplete="shipping zip-code"` and `autocomplete="billing zip-code"`.
```
Comment (chrisdavidmills) [inline files/en-us/web/html/attributes/autocomplete/index.md:30]:
```suggestion
Some tokens may be used more than once with potentially different expected values, such as the `zip-code` token in a form that contains both shipping and billing addresses. Including multiple different tokens in a space-separated list causes the associated form controls to be given unique autocomplete values: in this case, `autocomplete="shipping zip-code"` and `autocomplete="billing zip-code"`.
```

File: files/en-us/web/html/attributes/autocomplete/index.md [hunk 2]
```diff
@@ -11,17 +11,29 @@ The HTML `autocomplete` attribute lets web developers specify what if any permis
...
+
+Some autocomplete values may need to be re-used multiple times. For example, a form may contain multiple shipping addresses and therefore multiple occurrences of `"shipping zip-code"` while still expecting different values. In these cases, the first token in the space-separated list of tokens can be a `section-*` token; a token whose first eight characters are an ASCII case-insensitive match for the string "section-", meaning that the field belongs to the named group.
+
+If including the `autocomplete` attribute on {{HTMLElement("input/hidden", "hidden")}} input elements (`<input type="hidden">`), its value must be an ordered list of space-separated tokens; the `on` and `off` keywords are not allowed.
+
+The source of the suggested values is generally up to the browser; typically values come from past values entered by the user, but they may also come from pre-configured values. For instance, a browser might let the user save their name, address, phone number, and email addresses for autocomplete purposes. Perhaps the browser offers the ability to save encrypted credit card information, for autocompletion following an authentication procedure.
```
Comment (chrisdavidmills) [inline files/en-us/web/html/attributes/autocomplete/index.md:36]:
```suggestion
The source of the suggested values is generally up to the browser; typically values come from past values entered by the user, but they may also come from pre-configured values. For instance, a browser might let the user save their name, address, phone number, and email addresses for autocomplete purposes. The browser may also offer the ability to save encrypted credit card information, for autocompletion following an authentication procedure.
```

File: files/en-us/web/html/attributes/autocomplete/index.md [hunk 3]
```diff
@@ -11,17 +11,29 @@ The HTML `autocomplete` attribute lets web developers specify what if any permis
...
 > 3. The form to have a {{HTMLElement("input/submit", "submit")}} button
 
-{{EmbedInteractiveExample("pages/tabbed/attribute-autocomplete.html", "tabbed-shorter")}}
+If the same list of tokens is used in more than one form control, the user-agent will autocomplete all occurrences of the same `autocomplete` value with the same data value.
+
+Some tokens may be used more than once with potentially different expected values, such as the `zip code` token in a form that contains both shipping and billing addresses. By including multiple tokens in a space-separated list, the values are unique: in this case, `autocomplete="shipping zip-code"` and `autocomplete="billing zip-code"`.
+
+Some autocomplete values may need to be re-used multiple times. For example, a form may contain multiple shipping addresses and therefore multiple occurrences of `"shipping zip-code"` while still expecting different values. In these cases, the first token in the space-separated list of tokens can be a `section-*` token; a token whose first eight characters are an ASCII case-insensitive match for the string "section-", meaning that the field belongs to the named group.
```
Comment (chrisdavidmills) [inline files/en-us/web/html/attributes/autocomplete/index.md:32]:
```suggestion
Some autocomplete values may need to be re-used multiple times. For example, a form may contain multiple shipping addresses and therefore multiple occurrences of `"shipping zip-code"` while still expecting different values. In these cases, the first token in the space-separated list of tokens can be a `section-*` token. This token's first eight characters are always the string "section-", followed by a unique alphanumeric string chosen by the developer. All form fields given the same `section-*` token belong to a **named group**, and will therefore share the same autocomplete values.

 > **Note:** `section-*` values are matched case-insensitively. For example, `section-group1` denotes the same named group as `SECTION-GROUP1`.
```

Comment (chrisdavidmills) [reply files/en-us/web/html/attributes/autocomplete/index.md:32]: I'm not sure whether any of this is correct, but this is more the kind of complete explanation I'd hope for. A short example to show what a couple of fields in the same named group look like would also be quite helpful.

Feel free to correct and revise as makes sense ;-)

File: files/en-us/web/html/attributes/autocomplete/index.md [hunk 4]
```diff
@@ -32,7 +44,13 @@ If an {{HTMLElement("input")}}, {{HTMLElement("select")}} or {{HTMLElement("text
     > **Note:** In most modern browsers, setting `autocomplete` to "`off`" will not prevent a password manager from asking the user if they would like to save username and password information, or from automatically filling in those values in a site's login form. See [the autocomplete attribute and login fields](/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#the_autocomplete_attribute_and_login_fields).
 
 - "`on`"
+
   - : The browser is allowed to automatically complete the input. No guidance is provided as to the type of data expected in the field, so the browser may use its own judgement.
+
+- "`section-*`"
```
Comment (chrisdavidmills) [inline files/en-us/web/html/attributes/autocomplete/index.md:50]: The description here is fine, but in general, something feels off to me about this list of tokens. They are at the same level in the list as "off" and "on", but they are not complete values at the same level as those. They should be one level deeper, underneath a top-level description of "space-separated token list", or similar.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Generally fine, just a few bits for you to look at. Nice work, @estelle!

File: files/en-us/web/html/attributes/autocomplete/index.md (hunk 4 above)
Comment (estelle) [reply files/en-us/web/html/attributes/autocomplete/index.md]: I don't like the look, but i created nested DLs

**Review by estelle — COMMENTED** [review]
@chrisdavidmills  thanks for the review. updated everything.

File: files/en-us/web/html/attributes/autocomplete/index.md (hunk 4 above)
Comment (chrisdavidmills) [reply files/en-us/web/html/attributes/autocomplete/index.md]: The look of the nested DLs is never the greatest, but I do think this is easier to follow and makes more sense. Nice work.

**Review by chrisdavidmills — APPROVED** [review]
@estelle lovely, lovely, lovely.

**Pull request merged**